tests/test_browser.py

```diff
@@ -4480,7 +4480,7 @@ def test_webgl_simple_enable_extensions(self):
 
   @requires_graphics_hardware
   def test_webgpu_basic_rendering(self):
-    for args in [[], ['-s', 'ASSERTIONS'], ['-s', 'MAIN_MODULE=1']]:
+    for args in [[], ['-s', 'ASSERTIONS'], ['-s', 'ASSERTIONS', '--closure=1'], ['-s', 'MAIN_MODULE=1']]:
```

Comment:
Maybe just add this to the first iteration rather than running the test three times?